multiplexed-display.ino

```diff
@@ -78,6 +78,72 @@ void printContents() {
   }
 }
 
+void programmEEPROM() {
+	byte digits[] = { 0x7e, 0x30, 0x6d, 0x79, 0x33, 0x5b, 0x5f, 0x70, 0x7f, 0x7b };
+	byte minusSign = 0x01;
+
```

Comment:
let's define a white space for the display
byte blank = 0x00;

Then the code is more readable, where we compare d2 == 0, write **blank**.